pyvera/subscribe.py

```diff
@@ -64,16 +63,21 @@ def _event_device(self, device, device_data):
         sending = comment.find('Sending') >= 0
         if sending and state == STATE_NO_JOB:
             state = STATE_JOB_WAITING_TO_START
-        if (
+        if state == STATE_JOB_IN_PROGRESS and hasattr(device, 'is_lock'):
```

Comment:
To make it similar to the code above - perhaps just set the state if it's wrong.
Also if you check the class you can avoid needing to add an attribute.
Might as well remove the error trace now you've got it working!
````python
    if state == STATE_JOB_IN_PROGRESS and isinstance(devices[0], VeraLock):
        # VeraLocks don't complete - so force state
        state = STATE_JOB_DONE
````